stylesheets/utilities/variables.css

```diff
@@ -0,0 +1,62 @@
+:root{
+    --primary: #003d2b;
+    --primary-hover: #00704F;
+    --secondary:#3D0012;
+    --secondary-hover:#700021;
+
+    --success: #22bb33;
+    --success-hover:#36DA48;
+    --danger: #BB2124;
+    --danger-hover:#DB3437;
+
+    --white: #FFFFFF;
+    --white-gray: #f7f7f7;
+
+    --grey-lightest:#e4e4e4;
+    --grey-lighter:#c5c5c5;
+    --grey-dark:#727272;
+
+    --radius-sm: 5px;
+    --radius-md: 10px;
+    --radius-lg: 15px;
+    --radius-rounded: 999px;
+
+    --wgt-light: 300;
+    --wgt-regular: 400;
+    --wgt-bold-semi: 600;
+    --wgt-bold: 700;
+    --wgt-bold-dark: 900;
+
+    --size-0: 0;
```

Comment:
i think having these many values as css variable doesn't make much sense and also i don't see any pattern in the variable values . you can think of a better approach to this.

Reply:
Thanks Farhan, will try to update changes.